src/appleseed.studio/mainwindow/project/entityeditor.cpp

```diff
@@ -733,15 +738,14 @@ void EntityEditor::slot_open_file_picker(const QString& widget_name)
         const bf::path file_root_path = file_path.parent_path();
 
         QFileDialog::Options options;
-        QString selected_filter;
 
         QString filepath =
-            QFileDialog::getOpenFileName(
-                m_parent,
-                "Open...",
-                QString::fromStdString(file_root_path.string()),
-                filter,
-                &selected_filter,
+            get_open_filename(
+                m_parent,               // parent
+                "Open...",              // caption
+                filter,                 // filter
+                m_settings,           // settings
+                SETTINGS_FILE_DIALOG_ENTITIES,
```

Comment:
If `file_picker_type == "image"` then you should use `SETTINGS_FILE_DIALOG_OIIO_TEXTURES` instead. In other cases, `SETTINGS_FILE_DIALOG_ENTITIES` makes sense I think.